pkg/model/stack_serializer.go

```diff
@@ -38,6 +38,10 @@ type StackRaw struct {
 	Endpoints EndpointSpec               `yaml:"endpoints,omitempty"`
 	Volumes   map[string]*VolumeTopLevel `yaml:"volumes,omitempty"`
 
+	// Extensions
+
+	Extensions map[string]interface{} `yaml:",inline" json:"-"`
```

Comment:
If I get it correctly, this will parse anything not parsed by any other field.
Could you compare the output we are giving for current not supported fields with the output generated by this version? Right now we are using strict yaml parsing. To be honest, our current parsing errors are quite hard to understand, maybe we can do it better with this approach

Reply:
The only problem I see with the new errors management is that we can not show the line where the error is. But not showing model.StackRaw it's better because we are not giving any clue of what the error is
Previous version error:
 ```
x  Invalid stack manifest:
    - line 3: field random-field not found in type model.StackRaw
    See https://okteto.com/docs/reference/stacks/ for details
```

New error:
```
 x  random-field is not supported. More information is available here: https://okteto.com/docs/reference/stacks/
```

Reply:
@rberrelleza @melissa-okteto what do you think about the new error message for syntax errors in the okteto stack manifests?

Reply:
how does this compare with the errors you get in `docker compose up`?

Reply:
On compose the error they give is not very informative either:
```
ERROR: The Compose file './docker-compose.yml' is invalid because:
'random-field' does not match any of the regexes: '^x-'
```